docs/user-manual.md

```diff
@@ -115,48 +114,6 @@ To reboot a node without removing volumes, follow these steps:
 3. Run `kubectl uncordon NODE` after the node comes back online.
 4. After reboot, Pods will be rescheduled to the same node because PVCs remain intact.
 
-Inline Ephemeral Volumes (**deprecated**)
```

Comment:
Could you please write here what is written in the "About Generic Ephemeral Volume" section of this PR description?

Reply:
I added.
https://github.com/topolvm/topolvm/pull/494/commits/95f9e7974f34a61a2308e1e8f192f40372f71128